pkgs/os-specific/linux/netatop/netatop.service.patch

```diff
@@ -0,0 +1,7 @@
+--- a/netatop.service
++++ b/netatop.service
+@@ -11,3 +11,3 @@
+ ExecStopPost=@kmod@/bin/rmmod netatop
+-PIDFile=/var/run/netatop.pid
++PIDFile=/run/netatop.pid
```

Comment:
Those should be symlinked to one another.

Reply:
Yes, they are, at least on my system. However systemd warns about using /var/run and I wanted to suppress that warning.
I could live with the warning and remove that patch though, if you insist :-)

Reply:
Bonus points if you submit a patch upstream instead. Many distros warn about this by now. Fix it upstream: fix it everywhere.

Reply:
`/var/run -> /run` is actually not true in all instances. Atop supports many distros. The binaries do write to /var/run (hardcoded AFAICT). So changing it upstream breaks everywhere these are not symlinked.
In the end, it doesn't really matter, so I suggest to either leave it as is, and live with the systemd warning, or patch it like proposed.

Reply:
@pschyska thanks for letting me know that. Debian switched back in 2013 so I figured pretty much every distro must have switched over by now without ever having verified how valid this assumption was. I am curious which distros have not. Do you have any links I can read up on?

Back on topic... I'm not sure what to say here. I guess the patching is fine then. What do you think @SuperSandro2000?

Reply:
@aanderse Nothing to read up on, just a gut feeling that upstream wouldn't take the patch because it can't be assumed that these are symlinked in general.

Reply:
Oh. I thought you were saying there were some concrete example distros of this. In that case... even Debian switched back in 2013, and this is part of the FHS. Your gut feeling certainly could be correct, but what isn't correct would be software fighting against this change. :disappointed: 

> In FHS 3.0, /var/run is replaced by /run

Reply:
Could also be a substituteInPlace but a patch is also fine.

Reply:
@aanderse  https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s13.html "It is *valid* to implement /var/run as a symlink to /run." (emphasis mine). That said, `/var/run` is still defined to exist as to ` ensure compatibility with systems and software using an older version of this specification.`. So it's still "legal" to use `/var/run`, yet systemd's gonna systemd and decided to annoy the user with endless warnings ;-)